docs/graphql/manual/subscriptions/use-cases.rst

```diff
@@ -283,3 +283,83 @@ Now we can use the following subscription to display the latest poll result:
 
 Check this `sample app <https://realtime-poll.demo.hasura.app/>`__ for a working demo
 (`source code <https://github.com/hasura/graphql-engine/tree/master/community/sample-apps/realtime-poll>`__).
+
+How to get diffs via subscriptions
+------------------------------------------
+
+Subscriptions implement the ``live query`` semantics i.e. you can subscribe to any query and get the new result set anytime the underlying data changes. In case you are only interested in the ``diff`` of changes that happened, then you can implement a simple subscribe and fetch pattern. 
```

Comment:
```suggestion
Subscriptions implement the ``live query`` semantics i.e. you can subscribe to any query and get the new result set any time the underlying data changes. In case you are only interested in the ``diff`` of changes, you can implement a simple subscribe and fetch pattern. 
```